### ISSUE TITLE
Create the source as an existing PVC, but DV is in use and pods are all reporting errors, still running and stuck and freezing

### ISSUE THREAD

**Issue by 905049079** [kubevirt/containerized-data-importer#3435]
[root@host1 ~]# kubectl get dv
NAME           PHASE             PROGRESS   RESTARTS   AGE
cirros         Succeeded         100.0%                55m
cirros-clone   CloneInProgress   N/A                   9m39s
cirros2        Succeeded         100.0%                39m

[root@host1 ~]# kubectl get pod
NAME                         READY   STATUS    RESTARTS   AGE
cdi-upload-cirros-clone      1/1     Running   0          9m42s
virt-launcher-testvm-sqxll   3/3     Running   0          87m
virt-launcher-vm1-zx9mj      2/2     Running   0          30m

[root@host1 ~]# kubectl -n cdi logs --tail=10 cdi-deployment-69bf869699-fnwtg
{"level":"debug","ts":"2024-09-14T07:16:16Z","logger":"events","msg":"pod default/virt-launcher-vm1-zx9mj using PersistentVolumeClaim cirros","type":"Warning","object":{"kind":"PersistentVolumeClaim","namespace":"default","name":"cirros-clone","uid":"09076d82-59fc-44ca-ad68-ef4f7aed29a1","apiVersion":"v1","resourceVersion":"135533"},"reason":"CloneSourceInUse"}
{"level":"debug","ts":"2024-09-14T07:16:18Z","logger":"controller.clone-controller","msg":"reconciling Clone PVCs","PVC":{"name":"cirros-clone","namespace":"default"}}
{"level":"debug","ts":"2024-09-14T07:16:18Z","logger":"controller.clone-controller","msg":"can't create clone source pod, pvc in use by other pod","namespace":"default","name":"cirros","pod":"virt-launcher-vm1-zx9mj"}
{"level":"debug","ts":"2024-09-14T07:16:18Z","logger":"events","msg":"pod default/virt-launcher-vm1-zx9mj using PersistentVolumeClaim cirros","type":"Warning","object":{"kind":"PersistentVolumeClaim","namespace":"default","name":"cirros-clone","uid":"09076d82-59fc-44ca-ad68-ef4f7aed29a1","apiVersion":"v1","resourceVersion":"135533"},"reason":"CloneSourceInUse"}
{"level":"debug","ts":"2024-09-14T07:16:20Z","logger":"controller.clone-controller","msg":"reconciling Clone PVCs","PVC":{"name":"cirros-clone","namespace":"default"}}
{"level":"debug","ts":"2024-09-14T07:16:20Z","logger":"controller.clone-controller","msg":"can't create clone source pod, pvc in use by other pod","namespace":"default","name":"cirros","pod":"virt-launcher-vm1-zx9mj"}
{"level":"debug","ts":"2024-09-14T07:16:20Z","logger":"events","msg":"pod default/virt-launcher-vm1-zx9mj using PersistentVolumeClaim cirros","type":"Warning","object":{"kind":"PersistentVolumeClaim","namespace":"default","name":"cirros-clone","uid":"09076d82-59fc-44ca-ad68-ef4f7aed29a1","apiVersion":"v1","resourceVersion":"135533"},"reason":"CloneSourceInUse"}
{"level":"debug","ts":"2024-09-14T07:16:22Z","logger":"controller.clone-controller","msg":"reconciling Clone PVCs","PVC":{"name":"cirros-clone","namespace":"default"}}
{"level":"debug","ts":"2024-09-14T07:16:22Z","logger":"controller.clone-controller","msg":"can't create clone source pod, pvc in use by other pod","namespace":"default","name":"cirros","pod":"virt-launcher-vm1-zx9mj"}
{"level":"debug","ts":"2024-09-14T07:16:22Z","logger":"events","msg":"pod default/virt-launcher-vm1-zx9mj using PersistentVolumeClaim cirros","type":"Warning","object":{"kind":"PersistentVolumeClaim","namespace":"default","name":"cirros-clone","uid":"09076d82-59fc-44ca-ad68-ef4f7aed29a1","apiVersion":"v1","resourceVersion":"135533"},"reason":"CloneSourceInUse"}

version：1.60.3

**Comment by awels** [comment]
Yes this is expected behavior, we do not allow clones while the VM is running. It would make no sense to copy data while it is being modified by a running pod.
